### ISSUE TITLE
Delayed job not running with minConsumerCount(0)

### ISSUE THREAD

**Issue by ivashov** [yigit/android-priority-jobqueue#30]
I have the following setup

```
		Configuration configuration = new Configuration.Builder(this)
				.minConsumerCount(0)
				.maxConsumerCount(3)
				.build();
```

Job constructor
```
super(new Params(0).delayInMs(2000));
```

Job not running after 2 seconds.
Logs:
```
07-05 20:35:43.501  16161-16340/com.github.ivashov.jobqueuetest D/JOBS﹕ added job id: -2147483647 class: MyJob priority: 0 delay: 1000 group : null persistent: false requires network: false
07-05 20:35:43.501  16161-16340/com.github.ivashov.jobqueuetest D/MyJob﹕ onAdded
07-05 20:35:43.511  16161-16340/com.github.ivashov.jobqueuetest D/JOBS﹕ pool-1-thread-1: load factor check. false = (0 < 0)|| (0 * 3 < 0 + 0). consumer thread: false
```

**Comment by ivashov** [comment]
Complete project: https://github.com/ivashov/JobQueueIssue

**Comment by yigit** [comment]
Ok, I can reproduce with a unit test, looks bad, will check it. Thanks for the report.

**Comment by yigit** [comment]
Just fixed this, I'm planning to release 1.3.2 today so should be out soon.
